debian/changelog

```diff
@@ -1,3 +1,9 @@
+kano-profile (4.0.0-1) unstable; urgency=low
```

Comment:
This is not necessary since we haven't released 4.0 yet, no point in supporting `dist-upgrade` on our devel images. Please keep it under `4.0.0-0`.

Reply:
That will break upgrading that package for you guys who already have 4.0.0 installed. Are you sure you want me to do that?